react-native/package.json

```diff
@@ -24,8 +24,8 @@
   "main": "dist/sn-table.js",
   "peerDependencies": {
     "react": "18.2.0",
-    "react-native": "0.70.1",
-    "@nebula.js/stardust": "2.x"
+    "@nebula.js/stardust": "3.x",
```

Comment:
Sorry I was unclear, this needs to be `>=3.0.3` since we added `onContextMenu` to nebula in this version